src/services/connectors/mongo-connector.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+module.exports = function getMongoConnector(uri) {
+  return {
+    connect, // eslint-disable-line no-use-before-define
+  };
+
+  async function connect() {
+    return mongoose.createConnection(uri, { useNewUrlParser: true });
```

Comment:
in my opinion  , you need await,  not sure )

Reply:
it will be useless